js/ui/color_box/color_box.js

```diff
@@ -301,7 +301,7 @@ const ColorBox = DropDownEditor.inherit({
 
     _renderValue: function() {
         const value = this.option('value');
```

Comment:
const { value, editAlphaChannel } = this.option();
const convertToColor = value && editAlphaChannel;